project_template/webpack.config.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
   output: {
     path: out,
     filename: "[name].[hash].js",
-    publicPath: "./"
+    publicPath: "/"
```

Comment:
the relative path broke the `historyApiFallback` on the dev server, which I guess will also cause issues with Nightwatch if u want to jump straight to a particular URL when starting a test